cpp/oneapi/dal/algo/kmeans/backend/gpu/train_kernel_lloyd_dense_dpc.cpp

```diff
@@ -14,39 +14,40 @@
 * limitations under the License.
 *******************************************************************************/
 
+#include <tuple>
 #include <daal/src/algorithms/kmeans/kmeans_init_kernel.h>
-#include <src/algorithms/kmeans/oneapi/kmeans_dense_lloyd_batch_kernel_ucapi.h>
 
 #include "oneapi/dal/algo/kmeans/backend/gpu/train_kernel.hpp"
+#include "oneapi/dal/algo/kmeans/backend/gpu/kernels_integral.hpp"
+#include "oneapi/dal/algo/kmeans/backend/gpu/update_clusters.hpp"
+#include "oneapi/dal/algo/kmeans/backend/gpu/kernels_fp.hpp"
 #include "oneapi/dal/exceptions.hpp"
+#include "oneapi/dal/backend/primitives/ndarray.hpp"
 #include "oneapi/dal/table/row_accessor.hpp"
+#include "oneapi/dal/backend/transfer.hpp"
 #include "oneapi/dal/backend/interop/common_dpc.hpp"
 #include "oneapi/dal/backend/interop/error_converter.hpp"
 #include "oneapi/dal/backend/interop/table_conversion.hpp"
-#include "oneapi/dal/backend/transfer.hpp"
 
 namespace oneapi::dal::kmeans::backend {
 
-using std::int64_t;
 using dal::backend::context_gpu;
 using descriptor_t = detail::descriptor_base<task::clustering>;
 
-namespace daal_kmeans = daal::algorithms::kmeans;
 namespace daal_kmeans_init = daal::algorithms::kmeans::init;
 namespace interop = dal::backend::interop;
-
-template <typename Float>
-using daal_kmeans_lloyd_dense_ucapi_kernel_t =
-    daal_kmeans::internal::KMeansDenseLloydBatchKernelUCAPI<Float>;
+namespace pr = dal::backend::primitives;
+namespace de = oneapi::dal::detail;
```

Comment:
```suggestion
namespace de = dal::detail;
```

For consistency with the other aliases